pkg/monitors/ntp/ntp.go

```diff
@@ -0,0 +1,76 @@
+package ntp
+
+import (
+	"context"
+	"time"
+
+	"github.com/signalfx/golib/v3/datapoint"
+	"github.com/signalfx/signalfx-agent/pkg/core/config"
+	"github.com/signalfx/signalfx-agent/pkg/monitors"
+	"github.com/signalfx/signalfx-agent/pkg/monitors/types"
+	"github.com/signalfx/signalfx-agent/pkg/utils"
+	"github.com/sirupsen/logrus"
+
+	"github.com/beevik/ntp"
+)
+
+const minIntervalSeconds = 1800
+
+func init() {
+	monitors.Register(&monitorMetadata, func() interface{} { return &Monitor{} }, &Config{})
+}
+
+// Config for this monitor
+type Config struct {
+	config.MonitorConfig `singleInstance:"false" acceptsEndpoints:"false"`
+	// The host/ip address of the NTP server.
+	Host string `yaml:"host" default:"pool.ntp.org"`
```

Comment:
There should not be any default for this, it should be explicit every time this monitor is configured.

Reply:
ok I updated the code and seems good for me but I don't resolve this as I don't understand why the "required" value remains "no" in the doc https://github.com/xp-1000/signalfx-agent/blob/1299_ntp_monitor/docs/monitors/ntp.md may be do you have an idea?

Reply:
To make it required you just need to put a struct tag `validate:"required"`.

Reply:
oh my bad I already knew it .. sorry I just learned go in addition to signalfx integration process so I am not comfortable yet